tests/BenchmarkDotNet.IntegrationTests/TestCultureInfo.cs

```diff
@@ -0,0 +1,15 @@
+using System.Globalization;
+using BenchmarkDotNet.Helpers;
+
+namespace BenchmarkDotNet.IntegrationTests
+{
+    internal static class TestCultureInfo
+    {
+        public static readonly CultureInfo Instance;
+
+        static TestCultureInfo()
+        {
+            Instance = (CultureInfo) DefaultCultureInfo.Instance.Clone();
```

Comment:
what is the advantage of introducing this type if it's always using the same value as `DefaultCultureInfo`?

Reply:
In the future, we can easily mock it for testing special use cases without changing the original `DefaultCultureInfo`.